src/_includes/footer.html

```diff
@@ -1,18 +1,48 @@
-<hr>
-  <div class="row">
-      <div class="col-xs-12">
-          <footer>
-              <p class="text-center">
-                &copy; Copyright
-                <a href="http://www.apache.org">The Apache Software Foundation</a>,
-                {{ 'now' | date: "%Y" }}. All Rights Reserved.
-              </p>
-              <p class="text-center">
-                <a href="{{ site.baseurl }}/privacy_policy">Privacy Policy</a> |
-                <a href="{{ "/feed.xml" | prepend: site.baseurl }}">RSS Feed</a>
-              </p>
-          </footer>
+<footer class="footer">
+  <div class="footer__contained">
+    <div class="footer__cols">
+      <div class="footer__cols__col">
+        <div class="footer__cols__col__logo">
+          <img src="{{'/images/beam_logo_circle.svg'|prepend:site.baseUrl}}" class="footer__logo" alt="Beam logo">
```

Comment:
`baseUrl` is spelled wrong in all these – it should be `baseurl`. That may be the explanation for the many broken links and images.